examples/swarm/test_db.py

```diff
@@ -65,10 +65,10 @@ def testDbConnection(host, port, user, passwd):
     cursor.execute("INSERT INTO db_test VALUES ('testing123', 123)")
     cursor.execute("DROP TABLE IF EXISTS db_test")
     cursor.execute("DROP DATABASE IF EXISTS nupic_db_test")
-    return True
 
   except pymysql.err.OperationalError:
-    return False
+    print "Error: failed to connect to database"
+    raise pymysql.err.OperationalError
```

Comment:
This isn't right. We want to re-raise the original exception instance, including the message. Just use `raise` from within the exception handler block without any arg to do this.